test/stat_tracker_test.rb

```diff
@@ -269,16 +269,17 @@ def test_worst_season_by_team_id
 
   def test_worst_season_by_team_id_expanded
     skip
+    # THIS TEST IS BROKEN SOMEHOW
```

Comment:
Disregard this; it's unrelated to the method this PR is for, and I will address this `worst_season` method in a later PR.